src/style.css

```diff
@@ -1 +1,57 @@
 /* Styles go here */
+body {
+  margin: 0;
+}
+
+.container {
+  margin: 0;
+  display: flex;
+}
+
+.logo {
+  margin-top: 10px;
+  margin-left: 50px;
+  object-fit: none;
+  object-position: top;
+}
+
+.navigation {
+  width: 100vw;
+  font-family: "Roboto", sans-serif;
+  text-transform: uppercase;
+  font-size: 12px;
+}
+
+.list {
+  height: 60px;
+  line-height: 60px;
+  margin: 0 20px 0 0;
+  list-style-type: none;
+  cursor: pointer;
+}
+
+.nav_link {
+  text-decoration: none;
+  color: black;
+  padding: 20px 0;
+}
+
+.last {
+  margin-right: 30px;
+}
+
+.nav_link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  padding-bottom: 20px;
+  border-bottom: 4px solid #00acdc;
+}
+
+.fullList {
```

Comment:
Think this should have a more generic name, e.g. `navigation__list ` and then `<li>` elements, following to BEM can have class names e.g. `list__element` or `list__item`